### DISCUSSION TITLE
Howto Increase UMA size in MSI (AMI) Bios

### DISCUSSION THREAD

**Discussion by rohitnanda1443** [DavidS95/Smokeless_UMAF#77] [Q&A]
System: I have an MSI Pro B650-S WiFi board with Ryzen 8700G CPU. RAM: 64GB-5600mhz 

Question: 
1. Need to increase the UMA size to 32GB. The max the BIOS is enabling is 16GB
2. The EXPO / XMP setting to 1/2 does not help and the RAM still runs on 4800 mhz. How does one change it to 5600 mhz ?

**Comment by Blaubluter** [comment]
make .. never

**Comment by rohitnanda1443** [comment]
This is to use AI models in Ollama / vLLM and hence can easily max 16 if
one use a mistral.

**Reply by David112x** [reply]
AI models? on an iGPU? are you insane?
